packages/text-input-react/src/index.ts

```diff
@@ -1,3 +1,9 @@
-export { TextArea, Props as TextAreaPropsInterface } from "./TextArea";
-export { TextInput, Props as TextInputPropsInterface } from "./TextInput";
-export { BaseInputField, Props as BaseInputPropsInterface } from "./BaseInputField";
+import { TextArea, Props as TextAreaPropsInterface } from "./TextArea";
+import { TextInput, Props as TextInputPropsInterface } from "./TextInput";
+import { BaseInputField, Props as BaseInputPropsInterface } from "./BaseInputField";
+
+interface BaseInputProps extends BaseInputPropsInterface {};
+interface TextInputProps extends TextInputPropsInterface {};
+interface TextAreaProps extends TextAreaPropsInterface {};
```

Comment:
Ser ut som jeg gjorde feil i forslaget mitt 😅 Skal ikke være semikolon etter disse linjene.

Tipper `yarn lint --fix` tar det for deg hvis du vil

Reply:
Fixed 🤣